nmsg/input.c

```diff
@@ -468,6 +495,12 @@ input_open_stream_base(nmsg_stream_type type) {
 #else /* HAVE_LIBZMQ */
 		assert(type != nmsg_stream_type_zmq);
```

Comment:
From here to the end of the function, variable "type" is used but in the github.com viewer, it does not look like it is defined.

Reply:
A bit confused by this. Are you talking about ` input_open_stream_base(nmsg_stream_type type)` ?

Reply:
I think the code means to use `(*input)->type` as it does in some places

Reply:
I see what happened.  Github.com had multiple levels of code collapsed.  I was not seeing the start of input_open_stream_base()